src/qml/imports/Theme/QfProjectThumbnail.qml

```diff
@@ -13,6 +13,7 @@ Rectangle {
   property alias bgColor: thumbnailContentRectangle.color
   property alias showCustomizeIcon: customizeIcon.active
   property alias selected: selectedIndicator.active
+  property alias typeBadge: badge
```

Comment:
let's rename this to something else, like primaryBadge (since we'll have a secondaryBadge)